protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -48,6 +48,11 @@ func GenerateFile(p *protogen.Plugin, file *protogen.File) *protogen.GeneratedFi
 		genRegexVars(g, msg)
 		genValidateFunc(g, msg)
 		g.P()
+
+		for _, innerMsg := range msg.Messages {
```

Comment:
This will only work for one level deep of nested messages. I think this needs to be within the `genValidateFunc()` as a recursive function

Reply:
True. Let me work on that approach.